### DISCUSSION TITLE
How to build MT release lib

### DISCUSSION THREAD

**Discussion by nickychen123456** [openssl/openssl#23159] [Q&A]
I am currently using OpenSSL version 3.1.4 and would like to know how to build the multithreaded (MT) release version of the library. Are there any specific parameters to specify during the config phase to build the MT version of the library?

I currently have to manually modify the makefile by changing the line CNF_CFLAGS=/Gs0 /GF /Gy /M to CNF_CFLAGS=/Gs0 /GF /Gy /MT.

**Comment by nhorman** [comment]
openssl should build by default for a threaded environment.  Can you post the Configure command that you run and the ouput of:
```
./configdata.pm --dump
```
please
